lib/src/Base/Func/SpecFunc/openturns/SpecFunc.hxx

```diff
@@ -109,6 +109,11 @@ public:
   // Real number accuracy
   static const Scalar ScalarEpsilon;
 
+  // Information about capabilities
+  static Bool IsBoostAvailable();
+  static Bool IsMPFRAvailable();
+  static Bool IsMPCAvailable();
```

Comment:
could you also update python/test/t_features.py with each of these ?
it's useful to quickly run this script to know the available features enabled

Reply:
done